app/models/article.rb

```diff
@@ -490,6 +490,18 @@ def followers
     followers.uniq.compact
   end
 
+  def skip_indexing?
+    # should the article be skipped indexed by crawlers?
+    # true if unpublished, or spammy,
+    # or low score, not featured, and from a user with no comments
+    !published ||
+      (score < Settings::UserExperience.index_minimum_score &&
+       user.comments_count < 1 &&
+       !featured) ||
+      featured_number.to_i < 1_500_000_000 ||
+      score < -1
```

Comment:
is this `score < -1` the place we want to check index_minimum_score - not the longer "low score, author not active in comments, and not featured" check before?

I feel like we have a "any article with strongly negative score" guard that overrides the index minimum score, I don't think any admin would set the index minimum to -10 but if they did we'd have ignored them.